nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-processors/src/test/java/org/apache/nifi/processors/kafka/TestPutKafka.java

```diff
@@ -474,6 +475,18 @@ public void setStopFailingAfter(final Integer stopFailingAfter) {
         @Override
         public void close() {
         }
+
+        @Override
+        public void close(long arg0, TimeUnit arg1) {
+            // TODO Auto-generated method stub
+
+        }
+
+        @Override
+        public void flush() {
+            // TODO Auto-generated method stub
```

Comment:
As you can see, this is a MockProducer defined in test. Those two methods are only there to comply with the changes in Kafka interface and are not part of the test assertions.